src/formats/MOL2.cpp

```diff
@@ -70,18 +72,21 @@ void MOL2Format::read(Frame& frame) {
     frame.resize(0);
     frame.reserve(static_cast<size_t>(natoms));
 
-    auto junk = file_->readline();
+    // Skip a line
+    file_->readline();
+
+    // If charges are specified, we need to expect an addition term for each atom
     bool charges = (trim(file_->readline()) != "NO_CHARGES");
 
     while (!file_->eof()) {
-        auto curr_pos = file_->tellg();
-        auto line2 = trim(file_->readline());
+        const auto& curr_pos = file_->tellg();
```

Comment:
You don't need a reference here, std::streampos is not a big object, and it it returned by value by `tellg`